deployment/terraform/create.go

```diff
@@ -168,7 +168,7 @@ func (t *Terraform) Create(initData bool) error {
 
 		// Aurora sets the default_text_search_config parameter to 'simple';
 		// a more sane default for a generic deployment is 'english'
-		if t.config.TerraformDBSettings.InstanceEngine == "aurora-postgresql" {
+		if t.config.TerraformDBSettings.InstanceEngine == "aurora-postgresql" && t.output.HasDB() {
```

Comment:
I don't know why this if condition is inside `if t.output.HasAppServers() {`. It should be a standalone if condition, but I'll defer to Alejandro here.

Reply:
Agree, we could refactor this whole thing as something like:
```go
if t.output.HasAppServers{
	// ...
	if err := pingServer("http://" + url); err != nil {
		return fmt.Errorf("error whiling pinging server: %w", err)
	}
}

if t.output.HasDB() {
	if t.config.TerraformDBSettings.InstanceEngine == "aurora-postgresql" {
		// ...
	}
	if initData {
		// ...
	}
}
```